lms/envs/common.py

```diff
@@ -196,6 +196,7 @@
 
     # Toggle to enable certificates of courses on dashboard
     'ENABLE_VERIFIED_CERTIFICATES': False,
+    'DISABLE_HONOR_CERTIFICATES': False,
```

Comment:
May I request that some feature tag annotations be added here? Take a look at https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L126-L137 for an example. There is tooling being developed that will auto generate documentation based on the annotations. 

I would suggest values like the following, but feel free to change:

```
    # .. toggle_name: DISABLE_HONOR_CERTIFICATES
    # .. toggle_type: feature_flag
    # .. toggle_default: False
    # .. toggle_description: Set to True to disable honor certificates. Typically used when your installation only allows verified certificates, like courses.edx.org. 
    # .. toggle_category: certificates
    # .. toggle_use_cases: open_edx
    # .. toggle_creation_date: 2019-05-14
    # .. toggle_expiration_date: None
    # .. toggle_tickets: https://openedx.atlassian.net/browse/PROD-269
    # .. toggle_status: supported
```

Reply:
Sure, thank you for pointing it out @pdpinch!